### PR TITLE
Add code to serialize a trajectory to proto.

### PR DESCRIPTION
**Pull request by SirVer**

### CODE REVIEW

**Comment by Stonelinks** [comment]
Sure why not

**Comment by SirVer** [comment]
@Stonelinks ??

**Comment by Stonelinks** [comment]
Sorry, mistake on an auto reply!

File: cartographer/mapping/trajectory_node.cc [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+namespace cartographer {
+namespace mapping {
+
+proto::Trajectory ToProto(const std::vector<TrajectoryNode>& nodes) {
+  proto::Trajectory trajectory;
+  for (const auto& node : nodes) {
+    const auto& data = *node.constant_data;
+    auto* node_proto = trajectory.add_node();
+    node_proto->set_timestamp(common::ToUniversal(data.time));
+    *node_proto->mutable_pose_2d() =
```
Comment (damonkohler) [inline cartographer/mapping/trajectory_node.cc]: Just remove this from the proto. No need to keep the projected pose here.

**Comment by SirVer** [comment]
ptal.

**Pull request merged**